package.json

```diff
@@ -72,5 +72,7 @@
     "hooks": {
       "pre-commit": "lint-staged"
     }
-  }
+  },
+  "main": "index.js",
```

Comment:
main shouldn't be needed since this isn't a package
```suggestion
```

Reply:
Uhmmm
Thing I wasn’t sure how to  run the yarn fix
So I just did a few trials and the code went through 
So I am guessing I should just take out the  main right?

Reply:
Yes, this shouldn't affect script execution.

Reply:
Yes, I do need help, please.
for a clearer understanding of how this works

Reply:
Yeah, we don't need to have add this (though I suppose there's little harm in doing so).

To run `yarn fix`, you'll need to follow these steps:

1. [Install yarn](https://classic.yarnpkg.com/en/docs/install)
2. Navigate to the root directory of this project (where `package.json` is located)
3. Run `yarn` on the command line (or `yarn install`) to install the dependencies
4. Run `yarn fix` on the command line